File: api-examples/cat/search/js/spatial_search_circle.js
var settings = {
  "url": "https://<catalogue-server-ip>/catalogue/v1/search?lat=12.273737&lon=78.37475&radius=200000",
  "method": "GET",
  "timeout": 0,
};

$.ajax(settings).done(function (response) {
  console.log(response);
});

File: api-examples/cat/search/java/spatial_search_circle.java
OkHttpClient client = new OkHttpClient().newBuilder()
  .build();
Request request = new Request.Builder()
  .url("https://<catalogue-server-ip>/catalogue/v1/search?lat=12.273737&lon=78.37475&radius=200000")
  .method("GET", null)
  .build();
Response response = client.newCall(request).execute();
